_content/reference/realms_reading-faithfully.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Realms of Rebirth Taught by the Buddha"
+external_url: "https://readingfaithfully.org/realms-of-rebirth-taught-by-the-buddha/"
+drive_links:
+  - "https://drive.google.com/file/d/1DEStkYJjfCqkYcC4yrVvjKkIKWKEcihK/view?usp=sharing"
+course: cosmology
+tags:
+  - karma
+  - rebirth
+publisher: "Reading Faithfully"
```

Comment:
Reasonable, but let's stay consistent with [the other one](https://github.com/buddhist-uni/buddhist-uni.github.io/blob/main/_content/reference/pali-prounciation_rf.md?plain=1) where I made it the author